doc/nrf/libraries/others/profiler.rst

```diff
@@ -13,129 +13,196 @@ The output is provided using RTT and can be visualized in a custom Python backen
 
 See the :ref:`profiler_sample` sample for an example of how to use the Profiler.
 
+.. _profiler_configuration:
+
 Configuration
 *************
 
-Apart from standard configuration parameters, there is one required setting:
-
-:kconfig:`CONFIG_PROFILER`
-  Set this option to add the Profiler source code to the application.
-  If you use the Event Manager, you can also set this option by selecting :kconfig:`CONFIG_EVENT_MANAGER_PROFILER`.
+To use the Profiler for custom events, enable the :kconfig:`CONFIG_PROFILER` Kconfig option.
+This option adds the Profiler source code to the application.
 
 Call :c:func:`profiler_init` during the application start to initialize the Profiler.
-If you set :kconfig:`CONFIG_EVENT_MANAGER_PROFILER`, the Profiler is automatically initialized when you initialize the :ref:`event_manager`.
 
+Configuration for use with Event Manager
```

Comment:
Maybe we should mention this only in Event Manager Profiler documentation? To separate information about the integration from generic information about the profiler

Reply:
In profiler doc we could leave only link to Event Manager Profiler documentation (just to inform that Event Manager profiler exists and it can be used to profiler Event Manager).

Reply:
Ok, now I think it is better idea then what I have done.

Reply:
This was reworked. Please check after I commit shortly.